command/genesis/polybft_params.go

```diff
@@ -220,6 +220,11 @@ func (p *genesisParams) deployContracts(totalStake *big.Int) (map[types.Address]
 			artifact: contractsapi.L2StateSender,
 			address:  contracts.L2StateSenderContract,
 		},
+		{
+			// AccountAbstractionInvoker contract
```

Comment:
Unnecessary if the invoker is not deployed on genesis. (see other comment).

Reply:
Removed. Because I did re-base,  change is in first commit